config/initializers/diaspora_federation.rb

```diff
@@ -104,6 +104,17 @@
 
     on :receive_entity do |entity, sender, recipient_id|
       Person.by_account_identifier(sender).pod.try(:schedule_check_if_needed)
+      if User.exists?(recipient_id)
+        User.find(recipient_id).tap do |user|
+          next unless user&.person&.account_migration
+
+          Diaspora::Federation::Dispatcher.build(
+            user,
+            user.person&.account_migration,
```

Comment:
The `user.person` should also exist at this point, so the `&` is also redundant.